cmd/vela-server/server.go

```diff
@@ -97,7 +101,61 @@ func server(c *cli.Context) error {
 		return err
 	}
 
+	jitter := wait.Jitter(5*time.Second, 2.0)
+
+	logrus.Infof("retrieving initial platform settings after %v delay", jitter)
+
+	time.Sleep(jitter)
+
+	s, err := database.GetSettings(context.Background())
+	if s == nil || err != nil {
```

Comment:
instead of `err != nil`, should we check specifically for not found like we do here and then do some of the logic further down: https://github.com/go-vela/server/blob/8a744e48d224197478bb48a468186783b04c44db/api/dashboard/list_user.go#L70

Reply:
yeah agreed, ill tweak it

Reply:
went with this:
```
	ps, err := database.GetSettings(context.Background())
	if err != nil && !errors.Is(err, gorm.ErrRecordNotFound) {
		return err
	}

	// platform settings record does not exist
	if err != nil {
		logrus.Info("creating initial platform settings")
		......
```